graylog2-web-interface/src/views/components/searchbar/RefreshControls.tsx

```diff
@@ -83,30 +85,21 @@ class RefreshControls extends React.Component<Props> {
     }
   };
 
-  _onChange = (interval: number): void => {
-    RefreshActions.setInterval(interval);
-  };
-
-  _buttonLabel = (refreshConfigEnabled, naturalInterval) => {
-    let buttonText: React.ReactNode = 'Not updating';
-
-    if (refreshConfigEnabled) {
-      buttonText = <>Every {naturalInterval}</>;
-    }
-
-    return <ButtonLabel>{buttonText}</ButtonLabel>;
-  };
-
   render() {
     const { refreshConfig } = this.props;
+
+    const _onChange = (interval: number) => {
```

Comment:
No need to define this in `render()`, it can be pulled out of the component completely.